pyfar/dsp/filter/__init__.py

```diff
@@ -16,8 +16,8 @@
 )
 
 from .fractional_octaves import (
-    fractional_octave_bands,
-    reconstructing_fractional_octave_bands,
+    FractionalOctaveBands,
```

Comment:
Can you also bring back the imports for the old functions. Otherwise they won't be available for the user